src/material/paginator/paginator.scss

```diff
@@ -47,7 +47,7 @@ $button-icon-size: 28px;
   .mat-mdc-select {
     // The smaller font size inherited from the paginator throws off the centering of the select
     // inside the form field. This `line-height` helps to center it relative to the other text.
-    line-height: 1.5;
+    line-height: 2;
```

Comment:
1. This deviates from the design.
2. It'll be really difficult to land internally.

It might be easier if we go for an approach similar to the button where a transparent element provides a larger touch target.

Reply:
Reverted line-height change, added transparent touch target div to extend touch target size to 48 pixel height.